### PR TITLE
e2e: test browser medium group

### PR DESCRIPTION
**Pull request by pavloburykh**
Created test_browser medium group. Added first 3 tests.

### CODE REVIEW

**Comment by pavloburykh** [comment]
@churik the list of tests that have been split during refactoring:

**1. @marks.testrail_id(5368)
def test_change_log_level_and_fleet(self)**
 
was split to:

@marks.testrail_id(702178)
def test_profile_change_fleet(self) 

@marks.testrail_id(702165)
def test_profile_change_log_level(self)

**2. @marks.testrail_id(5424)
def test_open_url_with_non_english_text_connect_revoke_wallet_new_tab_open_chat_options(self)**

was split to:

@marks.testrail_id(702151)
def test_browser_open_url_with_non_english_text(self):

@marks.testrail_id(702180)
def test_browser_connect_revoke_wallet(self)

@marks.testrail_id(702181)
def test_browser_open_chat_options(self)

@marks.testrail_id(702182)
def test_browser_new_tab_open(self)

**3. @marks.testrail_id(5395)
def test_back_forward_refresh_navigation_history_kept_after_relogin(self)**

was split to:

@marks.testrail_id(702150)
def test_browser_back_forward_navigation_history_kept_after_relogin(self)

@marks.testrail_id(702179)
def test_browser_refresh_page(self)

File: test/appium/tests/atomic/test_public_chat_browsing.py [hunk 1]
```diff
@@ -509,6 +510,453 @@ def test_browser_web3_permissions_testdapp(self):
...
+        browsing.just_fyi('Navigate to next page and back')
+        browsing.element_by_text_part('може редагувати кожен').click()
+        browsing.element_by_text_part('написана спільно її читачами').wait_for_element(30)
+        browsing.browser_previous_page_button.click()
+        browsing.wait_for_element_starts_with_text('може редагувати кожен')
+
+        browsing.just_fyi('Relogin and check that tap on "Next" navigates to next page')
+        profile = self.home.profile_button.click()
+        # profile.switch_network()
+        profile.relogin()
```
Comment (churik) [inline test/appium/tests/atomic/test_public_chat_browsing.py]: profile.reopen_app() works faster, better to use it if re-login is not required explicitely

File: test/appium/tests/atomic/test_public_chat_browsing.py [hunk 2]
```diff
@@ -509,6 +510,453 @@ def test_browser_web3_permissions_testdapp(self):
...
+        self.home.dapp_tab_button.click()
+        browsing.open_tabs_button.click()
+        dapp.element_by_text_part(ua_url).click()
+        browsing.wait_for_element_starts_with_text('може редагувати кожен')
+        browsing.browser_next_page_button.click()
+        browsing.element_by_text_part('написана спільно її читачами').wait_for_element(30)
+
+        # browsing.open_tabs_button.click()
+        # browsing.close_all_button.click()
+
```
Comment (churik) [inline test/appium/tests/atomic/test_public_chat_browsing.py]: remove?

File: test/appium/tests/atomic/test_public_chat_browsing.py [hunk 3]
```diff
@@ -509,6 +510,453 @@ def test_browser_web3_permissions_testdapp(self):
...
+        dapp = self.home.dapp_tab_button.click()
+
+        dapp.just_fyi('Check non english text is shown in open url')
+        browsing = dapp.open_url('www.wikipedia.org')
+        for wiki_text in self.wiki_texts:
+            if not browsing.element_by_text_part(wiki_text).is_element_displayed(15):
+                self.errors.append("%s is not shown" % wiki_text)
+
+        # browsing.open_tabs_button.click()
+        # browsing.close_all_button.click()
```
Comment (churik) [inline test/appium/tests/atomic/test_public_chat_browsing.py]: remove?

File: test/appium/tests/atomic/test_public_chat_browsing.py [hunk 4]
```diff
@@ -509,6 +510,453 @@ def test_browser_web3_permissions_testdapp(self):
...
+        if not profile.element_by_text_part('custom_ropsten').is_element_displayed():
+            self.driver.fail("Network custom_ropsten was not added!")
+        profile.get_back_to_home_view()
+        # Switching back to Ropsten for the next cases
+        profile.switch_network('Ropsten with upstream RPC')
+
+        self.errors.verify_no_errors()
+
+    @marks.testrail_id(702164)
+    # TODO: Can be failed due to mailserver issue
```
Comment (churik) [inline test/appium/tests/atomic/test_public_chat_browsing.py]: not sure it is still relevant

**Review by churik — APPROVED** [review]
Excellent work @pavloburykh !

File: test/appium/tests/atomic/test_public_chat_browsing.py (hunk 1 above)
Comment (pavloburykh) [reply test/appium/tests/atomic/test_public_chat_browsing.py]: done

File: test/appium/tests/atomic/test_public_chat_browsing.py (hunk 2 above)
Comment (pavloburykh) [reply test/appium/tests/atomic/test_public_chat_browsing.py]: removed

File: test/appium/tests/atomic/test_public_chat_browsing.py (hunk 3 above)
Comment (pavloburykh) [reply test/appium/tests/atomic/test_public_chat_browsing.py]: removed

File: test/appium/tests/atomic/test_public_chat_browsing.py (hunk 4 above)
Comment (pavloburykh) [reply test/appium/tests/atomic/test_public_chat_browsing.py]: removed comment

**Pull request merged**